src/scheduler/pbs_sched_utils.cpp

```diff
@@ -0,0 +1,1204 @@
+/*
```

Comment:
In my opinion libsched.a is for scheduling policies.  The stuff that was in pbs_sched.c was separate.  Let's put this in its own library.